backend/corpora/lambdas/api/v1/curation/collections/common.py

```diff
@@ -38,6 +40,23 @@
 )
 
 
+def extract_doi_from_links(collection: dict):
+    """
+    Pull out the DOI from the 'links' array and return it along with the altered links array
+    :param collection: the Collection
+    :return: None
+    """
+    doi, resp_links = None, []
+    if links := collection.get("links"):
+        for link in links:
```

Comment:
Remove line 50, and do this:
```suggestion
        for link in collection.get("links", []):
```